kratos/python/add_processes_to_python.cpp

```diff
@@ -261,11 +261,11 @@ void  AddProcessesToPython(pybind11::module& m)
             ;
 
 
-    class_<AssignScalarVariableToConditionsProcess, Process>(m,"AssignScalarVariableToConditionsProcess")
+    class_<AssignScalarValueToConditionsProcess, Process>(m,"AssignScalarVariableToConditionsProcess")
             .def(init<ModelPart&, Parameters >())
     ;
 
-    class_<AssignScalarFieldToConditionsProcess , Process>(m,"AssignScalarFieldToConditionsProcess")
+    class_<AssignValueFieldToConditionsProcess , Process>(m,"AssignScalarFieldToConditionsProcess")
             .def(init<ModelPart&, Parameters >())
     ;
```

Comment:
Actually these are used in:
* kratos/python_scripts/assign_scalar_variable_to_conditions_process.py
* kratos/python_scripts/assign_vector_by_direction_to_condition_process.py

The second one is actually present in some json, so I assume is being used by:
* ShapeOptimizationApplication
* ContactStructuralMechanicsApplication

Reply:
I agree. This cannot be changed

Reply:
Well the python name is not changed, the class was only renamed because it presented conflicts with the original one. The original one is used for sure, the one in the core, I guess it is useless.